### DISCUSSION TITLE
Request: A way to mass load 20 games to fill out the Home list

### DISCUSSION THREAD

**Discussion by polo421** [SteamDeckHomebrew/decky-loader#663] [Ideas]
I feel like my recently played Home list is constantly filled with games I would rather not be there. For whatever reason. I was wondering if there could be a Decky plugin that would mass load 20 predetermined games that would then clear out your recent list with exactly the games you want to see. I'm not sure it's even possible but this seemed like a good place to mention it. Thanks.

**Comment by TrainDoctor** [comment]
We do not allow plugin requests on our GitHub pages. We have such a channel on our discord if the request is important enough for you.
